applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes.h

```diff
@@ -531,7 +531,12 @@ class TwoFluidNavierStokes : public FluidElement<TElementData>
 
     void AddSurfaceTensionContribution(
```

Comment:
Why is required to change the signature of this function to add the momentum term?

Reply:
I needed interface shape functions also for the correction term, therefore, took out the corresponding part of the code.

Reply:
OK, so you only compute them once right?

Reply:
Yes.